src/App.js

```diff
@@ -14,7 +15,8 @@ function App() {
                 <Switch>
                     <Route exact path={"/"} component={Home}/>
                     <Route exact path={"/shop"} component={MainShopPage}/>
-                    <Route exact path={'/item'} component={Item}/>
+                    <Route exact path={'/cloud'} component={Cloud}/>
+                    <Route exact path={'/unicorn'} component={Unicorn}/>
```

Comment:
maybe these could be `/shop/unicorn`
It doesn't matter that much but I think the `/shop/unicorn` format is more standard
and you could expand it in the future to `/shop/%%anyproduct%%` and make it dynamic

Reply:
i am going to use that idea.